packages/checkout/blocks-registry/index.ts

```diff
@@ -180,6 +180,18 @@ export const registerCheckoutBlock = (
 		registerExperimentalBlockType( blockName, {
 			...options.configuration,
 			category: 'woocommerce',
+			attributes: {
+				...options.configuration.attributes,
+				lock: {
+					type: 'object',
+					default: {
+						remove: !! options.configuration.force,
+						move:
+							options.configuration.attributes?.lock?.default
+								?.move,
+					},
+				},
+			},
```

Comment:
Add lock attribute to the block in case it's not already there, but making sure we don't override move lock. This needs more cleaning tomorrow.